skeleton/protractor/README.md

```diff
@@ -6,7 +6,7 @@ First, run `au run` and keep it running.
 
 Then run `au protractor`.
 
-To perform a test-run and reports the results, do `au protractor --run`.
+To perform a test-run in interactive mode, do `au protractor --interactive`.
```

Comment:
I guess you forgot updating doc.
```
To perform a test-run in headless mode, do `au protractor --headless`.
```

There is one more preference of `--interactive` in this doc.

Reply:
My bad, I thought I have changed that, nevertheless I pushed a new commit, I hope I did not miss anything else.